src/phyling/libphyling.py

```diff
@@ -104,23 +106,19 @@ def trim_gaps(
     """Trim the locations on MSA if the gappyness ratio (gaps/total length) is higher than a the arg gaps."""
     if gaps > 1 or gaps < 0:
         raise ValueError('The argument "gaps" should be a float between 0 to 1.')
-    infoList = [{"id": rec.id, "name": rec.name, "description": rec.description} for rec in pep_msa]
-    msa_array = np.array([list(rec) for rec in pep_msa])
-    gappyness = (msa_array == "-").mean(axis=0)
-    pep_trimList = np.where(gappyness > gaps)[0]
-    if pep_trimList.size > 0:
-        msa_array = np.delete(msa_array, pep_trimList, axis=1)
+    
+    clipkit_pep_msa = MSA.from_bio_msa(pep_msa)
```

Comment:
The default `gap_chars` is `None` for function `from_bio_msa`
https://github.com/JLSteenwyk/ClipKIT/blob/f4cb92a2c2248e21e3ecf161c1a1cd08152d4b16/clipkit/msa.py#L30
So probably need to assign a list of gap characters otherwise the nothing will be trimmed